public/app/plugins/panel/graph/series_overrides_ctrl.ts

```diff
@@ -1,160 +1,158 @@
 import _ from 'lodash';
 import angular from 'angular';
 
-export class SeriesOverridesCtrl {
+export function SeriesOverridesCtrl($scope, $element, popoverSrv) {
   /** @ngInject */
```

Comment:
the   /** @ngInject */ comment needs to be above the function (or constructor) to have any effect, now it's below, so this controller will not work in an minified & optimized build